spec/draft/design_topics/exceptions.rst

```diff
@@ -0,0 +1,28 @@
+.. _exceptions:
+
+Exceptions
+==========
+
+This standard specifies expected syntax and semantics for a set of APIs. When
+inputs to an API do not match what is expected, libraries may emit warnings,
+raise exceptions, or misbehave in unexpected ways. In general it is not
+possible to foresee or specify all the ways in which unexpected or invalid
+inputs are provided. Therefore, this standard does not attempt to specify
+exception or warning types to the extent needed in order to do exception
+handling in a portable manner. In general, it is expected that array library
+implementers follow `the guidance given by the documentation of the Python
+language <https://docs.python.org/3/library/exceptions.html>`__, and use either
+fitting builtin exception or warning types that are appropriate for the
+situation, or custom exceptions or warnings that derive from those builtin
```

Comment:
```suggestion
situation or use custom exceptions or warnings that derive from those builtin
```

Updated to ensure parallel construction. Otherwise, `use either` modifies "exception or warning types", which is not the intent.